tests/moe_ep/test_bf16_mxfp8_cutedsl_config.py

```
@@ -0,0 +1,373 @@
+        transform_buffer=transform_buffer,
+        accumulator_overlap=accumulator_overlap,
+        transform_k_tile=transform_k_tile,
+    )
+    assert config.num_experts_per_rank == 8
+
+
+def test_mixed_config_rejects_unsupported_implementation_and_token_back():
+    with pytest.raises(ValueError, match="implementation tuple"):
+        _config(mma_tiler_mnk=(256, 256, 128))
+    with pytest.raises(ValueError, match="standalone"):
+        _config(token_back_mode="standalone_warps")  # type: ignore[arg-type]
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "token_back_mode"

(S106)